linkerd/protocol/h2/src/main/scala/io/buoyant/linkerd/protocol/h2/grpc/GrpcClassifier.scala

```diff
@@ -127,9 +142,11 @@ object GrpcClassifiers {
   /**
    * a GrpcClassifier that marks any error in a specified list of codes
    * as ResponseClass.RetryableFailure, and the rest as ResponseClass.NonRetryableFailure
-   * @param retryableCodes a set of status codes which should be marked retryable
+   *
+   * @param retryableStatusCodes a set of status codes which should be marked retryable
+   * @param successStatusCodes a set of status codes which should be marked as success
    */
-  class RetryableStatusCodes(val retryableCodes: Set[Int]) extends GrpcClassifier {
-    override def retryable(status: GrpcStatus): Boolean = retryableCodes(status.code)
+  class RetryableStatusCodes(val retryableStatusCodes: Set[Int], override val successStatusCodes: Set[Int] = Set(0)) extends GrpcClassifier {
```

Comment:
for the sake of symmetry, it would be nice to do this the same way that we do for retryable: by overriding the def

Reply:
Looks good to me @adleong.  Thank you for your quick action on this matter.